biscuit-auth/samples/samples.json

```diff
@@ -2132,7 +2132,7 @@
           },
           "authorizer_code": "allow if true;\n",
           "revocation_ids": [
-            "365092619226161cf3973343f02c829fe05ab2b0d01f09555272348c9fcce041846be6159badd643aee108c9ce735ca8d12a009979c46b6e2c46e7999824c008"
+            "31f01031b8d9cc312c346674625a2218b19a126e931318dc02d8d53aec2775faecd3f848f60c1446573c37ea0664fefc768f35ba5ce4365b83ff06f0c0409b0e"
```

Comment:
this changes because the test uses the heterogeneous not equals, which is now a v3.3+ operator